src/coreclr/jit/gentree.cpp

```diff
@@ -21914,11 +21982,11 @@ GenTree* Compiler::gtNewSimdCmpOpNode(
 
         case GT_GE:
         {
-            if (IsBaselineVector512IsaSupportedOpportunistically())
+            if (compOpportunisticallyDependsOn(InstructionSet_EVEX))
```

Comment:
Same here, could be using the `canUseEvexEncoding` helper instead

Reply:
Changed.